packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -901,7 +902,60 @@ const Conversation = WebexPlugin.extend({
     this.webex.trigger('user-activity');
 
     return this.request(params)
-      .then((res) => res.body);
+      .then((res) => res.body)
+      .catch((error) => {
+        let convoUrl;
+
+        if (process.env.NODE_ENV === 'production') {
+          convoUrl = 'https://conv-a.wbx2.com/conversation/api/v1/conversations/';
+        }
+        else {
+          convoUrl = 'https://conversation-intb.ciscospark.com/conversation/api/v1/conversations/';
+        }
+        const transform = decryptTransforms.find((t) => t.name === 'decryptTextProp');
+        const contentTransform = decryptTransforms.find((t) => t.name === 'decryptContent');
```

Comment:
shouldn't the transformers defined in conversation plugin / index.js file handle the decryption already?

Reply:
decryption are defined in decrypt-transformers file. And the lost activity is already encrypted to resend it again it require to decrypt it first otherwise user will receive decrypt message.

Reply:
I think we should be able to resend using the same props/params as the original send?

Reply:
I tried but user receive encrypted message instead of original message. Therefore, I decrypted again.